MultiplayerMod/Multiplayer/State/PlayerState.cs

```diff
@@ -7,6 +7,7 @@ namespace MultiplayerMod.Multiplayer.State;
 public class PlayerState {
     public IPlayer Player { get; }
     public Vector2 CursorPosition { get; set; }
+    public bool Spawned { get; set; }
```

Comment:
From the usage perspective seems like it shows whether a player is spawned. Maybe WorldSpawned?
`playerState.Spawned` vs `playerState.WorldSpawned`